src/Makefile.qt.include

```diff
@@ -171,7 +171,8 @@ QT_MOC_CPP = \
 
```

Comment:
qt/macdarkmode.h needs to be added to the 
BITCOIN_QT_H
and 
QT_MOC_CPP
sections

Reply:
I'm not using any Qt classes, just regular QT_H is enough